deepspeed/comm/torch.py

```diff
@@ -20,13 +20,24 @@ def is_torch_two():
         return False
 
 
+def torch_ver_ge_1_13():
```

Comment:
This change seems to be independent of checkpointing. Is that correct?

Reply:
Right this is independent to checkpointing part, but found the issue along the for fixing checkpointing

Reply:
Sounds good. Thanks for the clarification!